packages/fxa-auth-server/lib/senders/emails/partials/appBadges/index.ts

```diff
@@ -3,18 +3,19 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 export const appBadges = `
+<mj-include path="./lib/senders/emails/css/appBadges/index.css" type="css" css-inline="inline" />
   <mj-section>
     <mj-group>
       <mj-column>
         <mj-image css-class="app-badges"
-          href="<%= iosUrl %>"
+          href="<%- iosUrl %>"
           src="https://accounts-static.cdn.mozilla.net/product-icons/apple-app-store.png" 
           alt="iOS badge">
         </mj-image>
       </mj-column>
       <mj-column>
         <mj-image css-class="app-badges"
-          href="<%= androidUrl %>"
+          href="<%- androidUrl %>"
```

Comment:
In ejs, `<%=` escapes the characters present in the string, because of which all the special characters present in the url were getting escaped whereas `<%-` renders the string as is

Reply:
This might be a good thing to add to the docs in this PR.